### PR TITLE
#866: add additional logging to deployment permission check

### PR DESCRIPTION
**Pull request by twschiller**
I can't reproduce #866, so add logging to try to replicate it on client machines after next release

### CODE REVIEW

File: src/utils/permissions.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ export async function requestPermissions(
   }
 
   // TODO: This only works in the Dev Tools; We should query the current or front-most window
-  // when this is missing in order to make it work in other contexts as well
+  //  when this is missing in order to make it work in other contexts as well
```
Comment (twschiller) [inline src/utils/permissions.ts:99]: IntelliJ treats indent as continuation of TODO text

Comment (fregante) [reply src/utils/permissions.ts:99]: While useful, I don’t think we should necessarily follow style conventions suggested by the IDE as you might end up having to fix these by yourself every time. 😅

File: src/options/pages/deployments/DeploymentBanner.tsx [hunk 2]
```diff
@@ -88,7 +88,7 @@ function useEnsurePermissions(deployments: Deployment[]) {
     }
 
     if (!accepted) {
-      addToast(`You declined the permissions`, {
+      addToast("You declined the permissions", {
```
Comment (fregante) [inline src/options/pages/deployments/DeploymentBanner.tsx:91]: Soon enough I’ll enable the XO config that enforces these minor things across the codebase 👍

File: src/utils/permissions.ts (hunk 1 above)
Comment (fregante) [reply src/utils/permissions.ts:99]: I mean if you like it you should put a ~ring~ *lint* on it.

**Review by fregante — APPROVED** [review]

File: src/utils/permissions.ts (hunk 1 above)
Comment (fregante) [reply src/utils/permissions.ts:99]: Or you can just apply these fixes as you see them, no problem either way if you’re ok with that.

Comment (twschiller) [reply src/utils/permissions.ts:99]: > I mean if you like it you should put a ring lint on it.

Haha, sadly a linter would have no way of knowing that the 2nd line is meant to be part of the TODO/FIXME in the preceding line. (Unless it used a heuristic like if the first line ended with a period)

File: src/options/pages/deployments/DeploymentBanner.tsx (hunk 2 above)
Comment (twschiller) [reply src/options/pages/deployments/DeploymentBanner.tsx:91]: I'm a bit surprised prettier can't fix these. Python Black IIRC won't convert unnecessary `f" " strings either

**Pull request merged**